packages/ra-core/src/controller/edit/useEditController.ts

```diff
@@ -220,7 +220,7 @@ export const useEditController = <
         isLoading,
         mutationMode,
         record,
-        redirect: DefaultRedirect,
+        redirect: redirectTo,
```

Comment:
This context variable isn't used anywhere in our code - I think it's a leftover from v3. I'll validate the PR, but I think we should ultimately mark it as deprecated.